packages/starknet-snap/src/rpcs/executeTxn.test.ts

```diff
@@ -90,41 +90,53 @@ const prepareMockExecuteTxn = async (
 describe('ExecuteTxn', () => {
   let callsExample: any;
 
-  it('executes transaction correctly if the account is deployed', async () => {
-    callsExample = callsExamples[0];
-    const {
-      account,
-      createAccountSpy,
-      executeTxnRespMock,
-      getEstimatedFeesSpy,
-      getEstimatedFeesRepsMock,
-      request,
-    } = await prepareMockExecuteTxn(
-      callsExample.hash,
-      callsExample.calls,
-      callsExample.details,
-      true,
-    );
+  it.each([
+    {
+      calls: callsExamples[0],
+      callStr: 'Call[]', // This is 'callStr'
```

Comment:
what does it mean by `callStr `
if think u are refer to the call object type?

i will prefer

testCaseTitle: 'an array of call object'
testCaseTitle: 'a call object'